burn-tensor/src/tensor/activation/base.rs

```diff
@@ -8,6 +8,13 @@ pub fn relu<const D: usize, B: Backend>(tensor: Tensor<B, D>) -> Tensor<B, D> {
     tensor.relu()
 }
 
+/// Applies the leaky rectified linear unit function.
+///
+/// f(x) = alpha * x for x < 0, f(x) = x for x >= 0
+pub fn leaky_relu<const D: usize, B: Backend>(tensor: Tensor<B, D>, alpha: f64) -> Tensor<B, D> {
+    tensor.leaky_relu(alpha)
```

Comment:
The backend function should be called instead:

```rust
B::leaky_relu(tensor, alpha)
```

Reply:
Done, except for the fact that I had to do some more conversions now.